
<html>
  <head>
    <style>

     h2 {
       text-align: center;
       font-family: Helvetica, Arial, sans-serif;
     }
     table { 
       margin-left: auto;
       margin-right: auto;
       table-layout: fixed;
     }
     table, th, td {
       border: 1px solid rgba(0,0,0,0.25);
       border-collapse: collapse;
     }
     th, td {
       padding: 5px;
       text-align: center;
       font-family: Helvetica, Arial, sans-serif;
       font-size: 80%;
     }
     table tbody tr:hover {
       background-color: #dddddd;
     }
     .wide {
       width: 100%; 
     }
     th.number, th.room {width: 5%;}
     th.title, th.authors {width:50%;}
     th.video { width: 5%; }

    </style>
  </head>
  <body>
    <table border="1" class="dataframe wide">
      <thead>
        <tr style="text-align: right;">
          <th class="number">Poster Number</th>
          <th class="room">Poster Room</th>
          <th>Title</th>
          <th>Authors</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>1</td>
          <td>A</td>
          <td>Assessing the importance of diagnostic information in learning low-dimensional embedding of high-dimensional abnormal neural correlates</td>
          <td>Wenjun Bai, Tomoki Tokuda, Okito Yamashita, Junichiro Yoshimoto</td>
        </tr>
        <tr>
          <td>2</td>
          <td>A</td>
          <td>Protein embeddings and deep learning predict binding residues for various ligand classes</td>
          <td>Maria Littmann, Michael Heinzinger, Christian Dallago, Konstantin Weissenow, Burkhard Rost</td>
        </tr>
        <tr>
          <td>3</td>
          <td>A</td>
          <td>Optimal Design of Stochastic DNA Synthesis Protocols Based on Generative Sequence Models (<a href="https://youtu.be/rMkGRsuEjcc">video</a>)</td>
          <td>Eli N Weinstein, Alan Nawzad Amin, Will Sussman Grathwohl, Daniel Kassler, Jean Disset, Debora Susan Marks</td>
        </tr>
        <tr>
          <td>4</td>
          <td>A</td>
          <td>Benchmarking deep generative models for diverse antibody sequence design</td>
          <td>Igor Melnyk, Payel Das, Vijil Chenthamarakshan, Aurelie Lozano</td>
        </tr>
        <tr>
          <td>5</td>
          <td>A</td>
          <td>Scalable estimation of microbial co-occurrence networks with Variational Autoencoders (<a href="https://youtu.be/lndAD-SOcLg">video</a>)</td>
          <td>James Morton, Justin Silverman, Gleb Tikhonov, Harri Lähdesmäki, Richard Bonneau</td>
        </tr>
        <tr>
          <td>6</td>
          <td>A</td>
          <td>CytoImageNet: A large-scale pretraining dataset for bioimage transfer learning (<a href="https://www.youtube.com/watch?v=qfEA-UR6tVM">video</a>)</td>
          <td>Stanley Bryan Zamora Hua, Alex Xijie Lu, Alan Moses</td>
        </tr>
        <tr>
          <td>7</td>
          <td>A</td>
          <td>Elucidation of proteasomal cleavage patterns from HLA peptidome data through deep learning</td>
          <td>Emilio Dorigatti, Julian Arnold, Bernd Bischl, Benjamin Schubert</td>
        </tr>
        <tr>
          <td>8</td>
          <td>A</td>
          <td>Bidirection Prediction Model for Transcriptomics and Proteomics</td>
          <td>Tianyu Liu, Yuge Wang, Hongyu Zhao</td>
        </tr>
        <tr>
          <td>9</td>
          <td>A</td>
          <td>AugmentedPCA: A Python Package of Supervised and Adversarial Linear Factor Models</td>
          <td>William E Carson IV, Austin Talbot, David Carlson</td>
        </tr>
        <tr>
          <td>10</td>
          <td>A</td>
          <td>A Graph Neural Network Approach to Molecule Carcinogenicity Prediction (<a href="https://youtu.be/14UKyHsTfso">video</a>)</td>
          <td>Philip Fradkin, Adamo Young, Lazar Atanackovic, Leo J Lee, Brendan Frey, BO WANG</td>
        </tr>
        <tr>
          <td>11</td>
          <td>A</td>
          <td>Organ Source Prediction for Exosomal Proteomics via Protein Language Models</td>
          <td>Xinbo Wu, Alexandru Hanganu, Ayuko Hoshino, Lav R. Varshney</td>
        </tr>
        <tr>
          <td>12</td>
          <td>A</td>
          <td>Enhancing the affinity of protein-protein interactions by multiple amino acid mutations predicted by deep neural networks</td>
          <td>Reut Moshe, Niv Papo, Yaron Orenstein</td>
        </tr>
        <tr>
          <td>13</td>
          <td>A</td>
          <td>Towards data-driven design of context-specific regulatory elements (<a href="https://youtu.be/GUJsXwR7lyg">video</a>)</td>
          <td>Peter Bromley, Wouter Meuleman</td>
        </tr>
        <tr>
          <td>14</td>
          <td>A</td>
          <td>Learning a representation of cellular morphology: unsupervised deep learning for shape and texture characterization of cells in EM data (<a href="https://youtu.be/a0Sc4WjJo9g">video</a>)</td>
          <td>Valentyna Zinchenko, Johannes Hugger, Virginie Uhlmann, Detlev Arendt, Anna Kreshuk</td>
        </tr>
        <tr>
          <td>15</td>
          <td>A</td>
          <td>Machine learning for novel antibiotic candidate domains - conjugated oligoelectrolytes</td>
          <td>Armi Tiihonen, Sarah J Cox-Vazquez, Guillermo Bazan, Tonio Buonassisi</td>
        </tr>
        <tr>
          <td>16</td>
          <td>A</td>
          <td>Bayesian nonparametric strategies for power maximization in rare variants association studies (<a href="https://youtu.be/aOim4H1N5tQ">video</a>)</td>
          <td>Lorenzo Masoero, Joshua Schraiber, Tamara Broderick</td>
        </tr>
        <tr>
          <td>17</td>
          <td>B</td>
          <td>Deep Generative Modeling of Transcription Factor-Gene Expression Relationships</td>
          <td>Byunguk Kang, Daniel Y. Zhu, William Souillard-Mandar, Hattie Chung, Fei Chen</td>
        </tr>
        <tr>
          <td>18</td>
          <td>B</td>
          <td>TeraTox empowers preclinical teratogenicity assessment with stem cells, sequencing, and explainable machine learning (<a href="https://youtu.be/QmoBpJVLbyo">video</a>)</td>
          <td>Jitao David Zhang, Manuela Jaklin, Nicole Schäfer, Nicole Clemann, Paul Barrow, Erich Küng, Lisa Sach-Peltason, Claudia McGinnis, Marcel Leist, Stefan Kustermann</td>
        </tr>
        <tr>
          <td>19</td>
          <td>B</td>
          <td>Optimal Design of Experiments for Simulation Based Inference of Mechanistic Acyclic Biological Networks (<a href="https://youtu.be/MPJxNKUvtgo">video</a>)</td>
          <td>Vincent Zaballa, Elliot Hui</td>
        </tr>
        <tr>
          <td>20</td>
          <td>B</td>
          <td>Protein Organization with Manifold Exploration and Spectral Clustering</td>
          <td>Shahab Shams, Geoffroy Dubourg-Felonneau, Eyal Akiva, Lawrence Lee</td>
        </tr>
        <tr>
          <td>21</td>
          <td>B</td>
          <td>Towards Remote Protein Homology Detection: Pairwise Alignment Using Locally Enriched Transformers</td>
          <td>Siva Muthupalaniappan, Sean R Eddy, Sam Petti</td>
        </tr>
        <tr>
          <td>22</td>
          <td>B</td>
          <td>A Bayesian nonparametric model for aligning spatial gene expression data (<a href="https://youtu.be/EnG7pBeXy-g">video</a>)</td>
          <td>Andrew Jones, F. William Townes, Didong Li, Barbara Engelhardt</td>
        </tr>
        <tr>
          <td>23</td>
          <td>B</td>
          <td>Leveraging Adversarial Reprogramming for Template-Constrained Protein Sequence Design</td>
          <td>Devleena Das, Inkit Padhi, Payel Das, Pin-Yu Chen, Amit Dhurandhar</td>
        </tr>
        <tr>
          <td>24</td>
          <td>C</td>
          <td>Deep Unsupervised Learning for Biosynthetic Gene Cluster Characterization (<a href="https://youtu.be/0dcf_eE9s7g">video</a>)</td>
          <td>Carolina Rios-Martinez, Nick Bhattacharya, Ava P. Soleimany, Lorin Crawford, Kevin K Yang</td>
        </tr>
        <tr>
          <td>25</td>
          <td>B</td>
          <td>Prediction of cell-cell communication directly from scRNA-seq latent spaces (<a href="https://youtu.be/HgMBcsyUXRc">video</a>)</td>
          <td>Tessa Durakis Green, Linus J Schumacher, Debora Susan Marks, Chris Sander</td>
        </tr>
        <tr>
          <td>26</td>
          <td>B</td>
          <td>InterDocker: End-to-End Cross-Attentive and Geometric Transformers for Efficient Iterative Protein Docking</td>
          <td>Allan Dos Santos Costa, Manvitha Ponnapati, Eric Alcaide, Kalyan Palepu, Suhaas M Bhat, Pranam Chaterjee, JOSEPH JACOBSON, Iddo Drori</td>
        </tr>
        <tr>
          <td>27</td>
          <td>B</td>
          <td>Prediction and design of biological sequences combining evolutionary sequence data with sparse labels (<a href="https://youtu.be/HSO-mC-6Awc">video</a>)</td>
          <td>Ada Shaw, Jung-Eun Shin, Debora Susan Marks</td>
        </tr>
        <tr>
          <td>28</td>
          <td>B</td>
          <td>Modeling gene regulatory network dynamics in tumor through graph embedding alignment (<a href="https://www.youtube.com/watch?v=3tuZpCCEY6E">video</a>)</td>
          <td>Hao Chen, Xiong Liu, Joseph Xu Zhou</td>
        </tr>
        <tr>
          <td>29</td>
          <td>B</td>
          <td>ProGSNN: Deep Multi-Scale Protein Representation Learning using Geometric Scattering</td>
          <td>Egbert Castro, Dhananjay Bhaskar, Jackson Grady, Smita Krishnaswamy</td>
        </tr>
        <tr>
          <td>30</td>
          <td>B</td>
          <td>cov2vec: encoding the genomic manifold of 2+ million SARS CoV2 viral sequences with protein language models. (<a href="https://www.youtube.com/watch?v=WFA0CeFncSg">video</a>)</td>
          <td>Salvatore Loguercio</td>
        </tr>
        <tr>
          <td>31</td>
          <td>B</td>
          <td>Protein language model embeddings for fast, accurate, alignment-free protein structure prediction</td>
          <td>Konstantin Weißenow, Michael Heinzinger, Burkhard Rost</td>
        </tr>
        <tr>
          <td>32</td>
          <td>B</td>
          <td>A generative model of the human proteome using across-species and within-species sequence data</td>
          <td>Jonathan Frazer, Mafalda Dias, Rose Orenbuch, Nikki Thadani, Kelly Brock</td>
        </tr>
        <tr>
          <td>33</td>
          <td>C</td>
          <td>MassFormer: Tandem Mass Spectrum Prediction with Graph Transformers (<a href="https://youtu.be/RAyenJS1224">video</a>)</td>
          <td>Adamo Young, Bo Wang, Hannes Röst</td>
        </tr>
        <tr>
          <td>34</td>
          <td>C</td>
          <td>Prediction of cell-cell communication in scRNA-seq latent spaces (<a href="https://youtu.be/HgMBcsyUXRc">video</a>)</td>
          <td>Tessa Durakis Green, Linus J Schumacher, Debora Susan Marks, Chris Sander</td>
        </tr>
        <tr>
          <td>35</td>
          <td>C</td>
          <td>Similarity Neural Networks for RBP Binding Site Detection</td>
          <td>Sepideh Saran, Mahsa Ghanbari, Uwe Ohler</td>
        </tr>
        <tr>
          <td>36</td>
          <td>C</td>
          <td>Joint embedding of sequence features (texts) and function labels (graphs) for protein function prediction</td>
          <td>Yue Cao, Yang Shen</td>
        </tr>
        <tr>
          <td>37</td>
          <td>C</td>
          <td>Self-Supervised Vision Transformers Learn Disentangled Representations in Histopathology</td>
          <td>Richard J. Chen, Rahul G Krishnan</td>
        </tr>
        <tr>
          <td>38</td>
          <td>C</td>
          <td>Contrastive VAE models to identify independent disease mechanisms in single-cell data (<a href="https://www.youtube.com/watch?v=RcuaqB8qoc0">video</a>)</td>
          <td>Atanasiu Demian, Harry Rose, Sam Abujudeh, Meltem Gurel</td>
        </tr>
        <tr>
          <td>39</td>
          <td>C</td>
          <td>Benchmarking algorithms to identify clinically-relevant sub-populations of patients (<a href="https://youtu.be/DhQ_xiohzY0">video</a>)</td>
          <td>Manuela Salvucci, Meltem Gürel, Sam Abujudeh, Marika Catapano, Gregor Lueg, Matyas Korom, Manav Leslie, Peter McErlean, Francesca Mulas</td>
        </tr>
        <tr>
          <td>40</td>
          <td>C</td>
          <td>Interpretable graph representation learning for lectin-glycan binding (<a href="https://youtu.be/OfsOxEdz_NM">video</a>)</td>
          <td>Joyce An, Somesh Mohapatra, Omar A Santiago-Reyes, Daria E Kim, Laura Kiessling, Rafael Gomez-Bombarelli</td>
        </tr>
        <tr>
          <td>41</td>
          <td>C</td>
          <td>Vaccine Design using Machine Learning of Human Degrons (<a href="https://www.youtube.com/watch?v=_lhlfRtAIYI">video</a>)</td>
          <td>Somesh Mohapatra, Nicholas L Truex, Mariane Bandeira Melo, Na Li, Wuhbet D Abraham, Jacob Joshua Lee Rodriguez, Darrell J Irvine, Bradley Pentelute, Rafael Gomez-Bombarelli</td>
        </tr>
        <tr>
          <td>42</td>
          <td>C</td>
          <td>Mapping Biology With a Unified Representation Space for Genomic and Chemical Perturbations to Enable Accelerated Drug Discovery (<a href="https://www.youtube.com/watch?v=L42kragUAAk">video</a>)</td>
          <td>Aurora S Blucher, Safiye Celik, James Douglas Jensen, James Taylor, Michael F Cuccarese, Jacob C Cooper, Jacob M Rinaldi, Carl Brooks, Michael A Statnick, Marta Fay, Nathan Lazar, Berton Earnshaw, Imran S Haque</td>
        </tr>
        <tr>
          <td>43</td>
          <td>C</td>
          <td>Simultaneous CUT&Tag profiling of the accessible and silenced regulome in single cells (<a href="https://youtu.be/QP9_Ld0YXo0">video</a>)</td>
          <td>Derek H. Janssens, Dominik Jenz Otto, Michael P. Meers, Kami Ahmad, Manu Setty, Steven Henikoff</td>
        </tr>
        <tr>
          <td>44</td>
          <td>C</td>
          <td>Rarity: a framework for the discovery of putative rare cell populations from single-cell imaging data</td>
          <td>Kaspar Märtens, Michele Bortolomeazzi, Francesca Ciccarelli, Christopher Yau</td>
        </tr>
      </tbody>
    </table>
  </body>
</html>
